Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -40,12 +40,19 @@ function OnUpdate_Door( furniture, deltaTime )
 		furniture.ChangeParameter("openness", deltaTime * -4)
 	end
 
-
 	furniture.SetParameter("openness", Clamp01(furniture.GetParameter("openness")) )
 
 	furniture.UpdateOnChanged(furniture);
 end
 
+function OnUpdate_Leak_Door( furniture, deltaTime )
+	furniture.tile.EqualiseGas(deltaTime * 10.0 * (furniture.GetParameter("openness") + 0.1))
```

Comment:
Should the `0.1` leak in normal doors be a parameter so that it'd be easy for other people creating doors to add more or less of it?